custom/champ/static/champ/js/knockout/prevision_vs_achievement_graph.js

```diff
@@ -0,0 +1,341 @@
+/* global ko, $, hqImport, moment, nv, d3 */
+
+ko.bindingHandlers.select2 = {
```

Comment:
Might be worth sharing this code between this file and the other two that use this binding.

Reply:
(not blocking)

Reply:
@orangejenny I moved this to the new file

Reply:
nice, thanks